### ISSUE TITLE
Mismatch between problem description and official solution - 789. Escape The Ghosts 

### ISSUE THREAD

**Issue by wangfys** [LeetCode-Feedback/LeetCode-Feedback#4580]
#### Your LeetCode-CN username
wangfys

#### Category of the bug
- [x] Question
- [x] Solution
- [ ] Language
- [ ] Missing Test Cases 

#### Description of the bug
I think the author of this problem didn't translate exactly what's in his/her mind to the description of the problem. As a result of this issue, the examples shown in the description are incorrect. The official solution on LeetCode-CN that using Manhattan Distance is also incorrect. I don't have a premium subscription so I don't know whether the official solution on LeetCode is the same.

Here is my reason for reporting this bug:

According to the description of the problem, there are several ghosts on the map. However, these ghosts are not described as enemies that they will try their best to make the player fail. On the contrary, the ghosts will independently choose to move 1 unit or stay still. They don't care about the player's action.

And the description says 'Return true if it is **possible** to escape, otherwise return false.' It doesn't says 'Return true if it is **always possible** to escape, otherwise return false.' There is a big gap between 'possible' and 'always possible'. **Possible** means if the ghosts choose a particular route that can't reach the player, the player successfully escaped. **Always possible** means whichever routes the ghosts choose to move, the player can always escape.

According to the current version of the description, if a ghost starts at [0,0], the player fails directly. Otherwise, the player can always win because there is a possibility that all ghosts incidentally move away so there is no burden between the start point and the destination.

#### Expected behavior
The example 2 should be:
Input: ghosts = [[1,0]], target = [2,0]
Output: true
Explanation: There is a possibility that the ghost moves to [1,1] when you move to [1,0]. Then the ghost stays still and you move to [2,0]. 

#### Recommendation for fix

Basically, I think this is just a mismatch between description and solution. So there are two ways to fix this bug:
1. Edit the description
2. Edit the official solution

I recommend editing the description from 'possible' to 'always possible'. So other parts all work fine.
Editing the official solution and the examples will make this problem meaningless. If the description is not edited, this problem is too easy.

**Comment by lc-kay** [comment]
Hi @wangfys 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-kay** [comment]
Hi @wangfys 
Thank you for your time. We've used your feedback to update the problem. We appreciate your support!
